tests/Services/SearchServiceTest.php

```diff
@@ -956,6 +1760,11 @@ public function redirectToProductPageOnDoSearchProvider(): array
                                 'barcodes' => [
                                     ['code' => '123123123'],
                                     ['code' => '321321321']
+                                ],
+                                'salesPrices' => [
```

Comment:
Yes, the test fails then, because we've added a price condition in the \Findologic\Services\SearchService::getVariationIdForRedirect method. I can't reproduce the way, that item doesn't have a salesPrice set.